wrapper/src/main/java/software/amazon/jdbc/util/ConnectionUrlParser.java

```diff
@@ -188,6 +192,42 @@ public static void parsePropertiesFromUrl(final String url, final Properties pro
       final String currentParameterName = param.substring(0, pos);
       props.setProperty(currentParameterName, currentParameterValue);
     }
+
+    // try to detect "wrapperCaseSensitive" parameter
+    String caseSensitiveSettingStr = props.getProperty(PropertyDefinition.CASE_SENSITIVE.name);
+    if (caseSensitiveSettingStr == null) {
+      // try low case
+      caseSensitiveSettingStr = props.getProperty(PropertyDefinition.CASE_SENSITIVE.name.toLowerCase());
+    }
+    if (caseSensitiveSettingStr == null) {
+      caseSensitiveSettingStr = PropertyDefinition.CASE_SENSITIVE.defaultValue;
+    }
+
+    final boolean caseSensitive = Boolean.parseBoolean(caseSensitiveSettingStr);
```

Comment:
Instead of having this here, what do you think of overloading the `getBoolean` method in `AwsWrapperProperty` and move this logic there?
E.g.

```java
  public boolean getBoolean(final Properties properties) {
    return this.getBoolean(properties, false);
  }

  public boolean getBoolean(final Properties properties, final boolean checkLowerCase) {
    Object value = properties.get(name);
    if (checkLowerCase && value == null) {
      value = properties.get(name);
    }
    if (value instanceof Boolean) {
      return (Boolean) value;
    }
    return Boolean.parseBoolean(properties.getProperty(name, defaultValue));
  }
```

Reply:
Then we can just simplify this to 
```java
final boolean caseSensitive = PropertyDefinition.CASE_SENSITIVE.getBoolean(props, true);
```